src/managers/KeybindManager.cpp

```diff
@@ -782,6 +785,14 @@ SDispatchResult CKeybindManager::handleKeybinds(const uint32_t modmask, const SP
 
             m_passPressed = sc<int>(pressed);
 
+            // We only process the releaseinputcapture dispatcher when input capture is active
```

Comment:
do this inside ::releaseInputCapture. It's ugly and cluttered here. Just return an error when it's not captured.

Reply:
Ive simplified the if, tell me if its good for u